### PR TITLE
UI/UX adjustments and fixes

### PR DESCRIPTION
**Pull request by ghengeveld**
Moved around a bunch of UI elements and tweaked many other things.

<details>
  <summary>📦 Published PR as canary version: <code>0.0.117--canary.148.07c98ce.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.117--canary.148.07c98ce.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.117--canary.148.07c98ce.0
  ```
</details>

### CODE REVIEW

File: src/Panel.tsx [hunk 1]
```diff
@@ -61,10 +61,12 @@ export const Panel = ({ active, api }: PanelProps) => {
   const [createdProjectId, setCreatedProjectId] = useState<Project["id"]>();
 
   if (gitInfoError) {
+    // eslint-disable-next-line no-console
+    console.error(gitInfoError);
     return (
       <Provider key={PANEL_ID} value={client}>
         <Sections hidden={!active}>
-          <GitNotFound gitInfoError={gitInfoError} />
+          <GitNotFound gitInfoError={gitInfoError} setAccessToken={setAccessToken} />
```
Comment (weeksling) [inline src/Panel.tsx:69]: setAccessToken doesn't seem to be a prop for GitNotFound.

File: src/screens/VisualTests/graphql.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export const FragmentLastBuildOnBranchBuildFields = graphql(/* GraphQL */ `
     }
     ... on CompletedBuild {
       result
-      testsForStatus: tests(statuses: $testStatuses) {
+      testsForStatus: tests(first: 1000, statuses: $testStatuses) {
```
Comment (weeksling) [inline src/screens/VisualTests/graphql.ts:56]: Does this limit the number of results? I would think we'd always want all test results for the build

File: src/utils/useTests.ts [hunk 3]
```diff
@@ -1,19 +1,35 @@
...
 
-import { BrowserInfo, StoryTestFieldsFragment, TestMode } from "../gql/graphql";
+import { BrowserInfo, StoryTestFieldsFragment, TestMode, TestStatus } from "../gql/graphql";
 
 type BrowserData = Pick<BrowserInfo, "id" | "key" | "name">;
 type ModeData = Pick<TestMode, "name">;
 
+const useGlobalValue = (key: string) => {
+  try {
+    return useGlobals()[0][key];
```
Comment (weeksling) [inline src/utils/useTests.ts:11]: Is there something special about the first global that we use it as a store?

File: src/utils/useTests.ts [hunk 4]
```diff
@@ -1,19 +1,35 @@
...
-    tests[0]?.comparisons[0]?.browser.id
-  );
-  const [selectedModeName, onSelectModeName] = useState<ModeData["name"]>(tests[0]?.mode.name);
+  const theme = useGlobalValue("theme");
+  const [selectedBrowserId, onSelectBrowserId] = useState<BrowserData["id"]>(() => {
+    const test = tests.find(({ status }) => status !== TestStatus.Passed) || tests[0];
+    return test?.comparisons[0]?.browser.id;
+  });
+  const [selectedModeName, onSelectModeName] = useState<ModeData["name"]>(() => {
+    const changed = tests.filter(({ status }) => status !== TestStatus.Passed);
```
Comment (weeksling) [inline src/utils/useTests.ts:28]: QQ: Are these tests limited to `testsForStory`?

**Review by weeksling — APPROVED** [review]
lgtm - big improvement in ui! 🚀

File: src/utils/useTests.ts (hunk 3 above)
Comment (ghengeveld) [reply src/utils/useTests.ts:11]: `useGlobals` returns a tuple of `[value, setValue]`. I just want the value here.

File: src/utils/useTests.ts (hunk 4 above)
Comment (ghengeveld) [reply src/utils/useTests.ts:28]: Yes, but `testsForStory` is filtered by `storyId` and returns max 100 results. It's unlikely that anyone would have that many tests for one story though. That's only possible if you have >100 modes/viewports. Seems unlikely.

**Pull request merged**